rel/overlay/etc/vm.args

```diff
@@ -61,6 +61,18 @@
 # Dirty IO schedulers are used for file IO.
 +SDio 16
 
+# Increase distribution buffer size from default of 1MB to 32MB. The default is
+# usually a bit low on busy clusters. Has no effect for single-node setups.
+# The unit is in kilobytes.
++zdbbl32768
```

Comment:
I think this line is missing a space (between `+zbbl` and the value).  I can imagine that the actual syntax does not require this space to be there but it would definitely contribute to better readability, as well as it would make more consistent with the formatting of other parameters in this file.

Reply:
I just kept the same syntax as we use a Cloudant

Reply:
Updated it have a space

Reply:
Cool, thank you.